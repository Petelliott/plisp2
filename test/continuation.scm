(println (call/cc (lambda (cont)
                    5
                    7)))

(println (call/cc (lambda (cont)
                    5
                    (cont 6)
                    7)))

;(define c #f)
;
;(println (call/cc (lambda (cont) (set! c cont))))
;
;(when c
;  (c 66)
;  (set! c #f))
