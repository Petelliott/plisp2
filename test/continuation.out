7
6
#<unspecified>
66